src/routes/city-detail.jsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import aqCheck from '../utils/aq-check';
+import './styles/city-detail.styles.scss';
+
+function CityDetail() {
+  const {
+    name, aqi, time, forecast,
+  } = useSelector((state) => state.cityInfo.cityDetail);
+  const { title, color } = aqCheck(aqi);
+
+  return (
+    <div className="city-container">
+      <div className="bar-head">
+        <Link to="/cities">
+          <svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth={1.5} stroke="currentColor" className="w-6 h-6 back-icon">
+            <path strokeLinecap="round" strokeLinejoin="round" d="M15.75 19.5L8.25 12l7.5-7.5" />
+          </svg>
+        </Link>
+        <span>AIR QUALITY DETAIL</span>
+      </div>
+      <div className="detail-body">
+        <div className="ct-header">
+          <h3>{name}</h3>
+        </div>
+        <div className="ct-info">
+          <div className="aqi">
+            <span>{time.s}</span>
+            <span className="aqi-n" style={{ backgroundColor: color }}>{aqi}</span>
+            <span className="aq-title">{title}</span>
+          </div>
+          <div className="forecast">
+            <h4>PM25 Forecast</h4>
+            <div className="fc-table">
+              <div className="fc-head">
+                <span>Date</span>
+                <span>avg</span>
+                <span>min</span>
+                <span>max</span>
+              </div>
+              {
+                forecast.map((item) => (
+                  <div key={item.id} className="pm25">
+                    <span>{item.day}</span>
+                    <span>{item.avg}</span>
+                    <span>{item.min}</span>
+                    <span>{item.max}</span>
+                  </div>
+                ))
```

Comment:
- Good job done so far, but I noticed the styling of your app is inconsistent. The background color for this page is not consistent with your other pages, Please fix this 🙏 
Your project:
![ydyd](https://user-images.githubusercontent.com/73607512/225968429-cfcbc5e8-ed32-4687-8049-e9197c61f666.PNG)

Reply:
Impressive work!!! 👏🏾. The design is consistent and great work balancing the different color shades 😍 

Homepage | Details page
------------- | -------------
![image](https://user-images.githubusercontent.com/38088818/226092556-8c00bcd8-8bdc-4bb3-b35c-9263226d476d.png) | ![image](https://user-images.githubusercontent.com/38088818/226092599-b43ee02e-3a11-425c-a250-2172d39218aa.png)

## OPTIONAL

It would be great if you fixed the overlapping categories content on the mobile view.

- Kindly use flexible CSS width units like `%` or `vw` to make sure that the width of the category square is automatically adjusted when the view width decreases.
- Kindly make sure that the categories form 2 columns on mobile view, your app shows 3 columns which is another reason why the content is overlapping outside the main container.

![image](https://user-images.githubusercontent.com/38088818/226092926-2c0546eb-a21a-4053-9067-23b33f47174e.png)

Reply:
@KDlamini Thanks for your feedback, I will do it.